project/app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,164 @@
+    <string name="tls" translatable="false">TLS</string>
+    <string name="preferencesSecurity">Sécurité</string>
+    <string name="accept">J\'accepte</string>
+    <string name="preferencesIdentification">Identification</string>
+    <string name="preferencesParameters">Paramètres</string>
+    <string name="preferencesKeepalive" translatable="false">Keepalive</string>
+    <string name="seconds">Secondes</string>
+    <string name="preferencesProfileId">Mode</string>
+    <string name="mode_mqtt_private_label">MQTT privé</string>
+    <string name="aLocation">Région</string>
+    <string name="events">Événements</string>
+    <string name="title_activity_regions">Régions</string>
+    <string name="title_activity_region">Région</string>
+    <string name="title_activity_map">Map</string>
+    <string name="title_activity_contacts">Copains</string>
+    <string name="title_activity_status">Statut</string>
+    <string name="title_activity_preferences">Préférences</string>
+    <string name="title_exit">Quitter l\'application</string>
+    <string name="unableToCopyCertificate">Impossible de copier le certificat sur le stockage interne</string>
+    <string name="preferencesCategoryNotificationOngoing">Notification en cours</string>
+    <string name="preferencesCategoryNotificationBackground">Notifications en arrière-plan</string>
+    <string name="preferencesCategoryAdvancedServices">Services</string>
+    <string name="preferencesCategoryAdvancedLocator">Localisateur</string>
+    <string name="preferencesCategoryAdvancedEncryption">Chiffrement</string>
+    <string name="preferencesCategoryAdvancedMisc">Misc</string>
+    <string name="contactDetailsTrackerId" translatable="false">Tracker ID</string>
+    <string name="exportWaypointsToEndpoint">Envoyer POIs</string>
+    <string name="preferencesExportFailed">Échec de l\'exportation des préférences</string>
+    <string name="preferencesExportSuccess">Exportation réussie</string>
+    <string name="welcomeFixIssue">Autoriser</string>
+    <string name="play_services_now_available">Les services Google Play sont maintenant disponibles</string>
+    <string name="play_services_not_available">Les services Google Play ne sont pas disponibles</string>
+    <string name="preferencesUrl" translatable="false">URL</string>
+    <string name="mode_http_private_label">HTTP privé</string>
+    <string name="transitionEntering">entré</string>
+    <string name="transitionLeaving">sorti</string>
+    <string name="errorPreferencesImportFailed">L\'importation a échoué</string>
+    <string name="welcome_play_heading">Services Google Play</string>
+    <string name="welcome_play_description">Owntracks nécessite les services Google Play</string>
+    <string name="icon_description_warning">Attention</string>
+    <string name="welcome_heading">Bienvenu</string>
+    <string name="welcome_description">Les étapes suivantes vous aideront à démarrer avec Owntracks</string>
```

Comment:
`s/Owntracks/OwnTracks/g`

Reply:
Will change this in all the `strings.xml`